pages/content/posts/swag-at-tpac-anaheim.md

```diff
@@ -0,0 +1,38 @@
+
+The SWAG CG was formed by [Dan Appelquist](https://mastodon.social/@torgo) in June 2024 and has been meeting weekly since then. Its mission is:
+
+> To increase the overall security of web application development, thereby making the web a more secure platform for web users. It will accomplish this by writing security best practices for web developers and providing a platform for stakeholder collaboration.
+
+Open Web Docs participates in this group because we think better documentation can improve the state of web security, and we think the SWAG CG can reveal gaps in web security documentation and provide technical support for us as we work to fill them.
+
+At TPAC [we presented four topics in security documentation](https://wbamberg.github.io/web-security-w3c-breakouts-september-2024/Templates/Overview.html) that have come up through the work of the SWAG CG:
+
+- **Security 101**: a page, or a series of pages, on [MDN](https://developer.mozilla.org), listing fundamental things a developer can do, that have a big impact on the security of the site. This is partly addressed by the work [Chris Mills](https://github.com/chrisdavidmills/) has done on [Practical security implementation guides](https://developer.mozilla.org/en-US/docs/Web/Security/Practical_implementation_guides), but there may be further updates needed.
+
+- **Security considerations for Web APIs**: some web platform features — [`fetch()`](https://developer.mozilla.org/en-US/docs/Web/API/Window/fetch), for example — have special implications for web security. Features like this should have a "Security considerations" section in their reference pages summarizing these implications, so they're more apparent to a developer trying to use the feature.
+
+- **Frameworks and libraries**: security documentation should help developers make good choices about which third-party software to use, either by recommending specific libraries or by [writing guidelines that developers can use to make their own choices](https://github.com/w3c-cg/swag/issues/1).
+
+- **Content security policy (CSP) adoption**: can more consistent and approachable documentation help more developers implement an effective [content security policy](/en-US/docs/Web/HTTP/CSP) for their sites? While [OWASP recommends a strict CSP](https://cheatsheetseries.owasp.org/cheatsheets/Content_Security_Policy_Cheat_Sheet.html#csp-types-granularallowlist-based-or-strict) which uses nonces and hashes rather than an allowlist, MDN's documentation does not mention allowlists. We should aim for consistency here so we can give a clear message to developers.
```

Comment:
should we define nonce? should we create a glossary entry? and allowlist?

Reply:
Well, the OWASP link just before this should help explain things. MDN doesn't currently have great documentation on this (which is one of the things we need to fix) so we don't really have a good target for it right now.